client/src/Palette.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+function Palette(props) {
+  const [paletteItems, setPaletteItems] = React.useState();
+
+  React.useEffect(() => {
+    fetch(`/profiles`)
+      .then((res) => res.json())
+      .then((data) => {
+        setPaletteItems(data.palettes);
+      });
+  }, []);
+
+  return (
+    <div className="flex">
+      {paletteItems &&
+        paletteItems.map((palettes, index) => (
+          <div className="relative m-3" key={index}>
```

Comment:
According to the react docs, `index` should only be used as a key as a last resort. Ideally you want a unique identifier like an `ID` from the DB. Maybe we can add a TODO comment here to add a better `key` once we have a real DB connection

Reply:
https://reactjs.org/docs/lists-and-keys.html#basic-list-component